src/main/java/elevator/Elevator.java

```diff
@@ -0,0 +1,17 @@
+package elevator;
+
+public interface Elevator{
+ Integer floor();
+ Boolean call();
```

Comment:
@dwaynewade0033 
1. What is returning value?
2. Which floor the elevator is being called when this method is invoked?